api/util/logging-helpers.js

```diff
@@ -0,0 +1,41 @@
+const { LOGGING_API_URL } = require('../config/config');
+const axios = require('axios');
+
+/**
+ * Add a new error by calling on the ErrorLog API.
+ * @param {Object} newError - The error that is to be added
+ * @param {(string|undefined)} errorToAdd.userEmail - The email of the user
+ *                                                  who has sent this error
+ * @param {(string|undefined)} errorToAdd.errorTime- The time the error occured
+ * @param {string} errorToAdd.apiEndpoint - The location of the error
+ * @param {string} errorToAdd.errordescription - The description of the error
+ */
```

Comment:
To have consistency with the comments for the function below, maybe add an `@return` comment for errorSaved to describe what it does

Reply:
yes thank you for catching this. adding it now.